libbeat/kibana/testdata/extensive/fields.yml

```diff
@@ -31,6 +31,11 @@
       description: >
         The timestamp when the event log record was generated.
 
+    - name: blob
+      type: binary
+      description: >
+        Some binary blob.
```

Comment:
I recommend to change the `fields.yml` file rather than the `extensive/fields.yml` as it is easier to grasp. 
The extensive tests were added to ensure that no breaking change was introduced when rewriting the code in go and it reflects the complete fields.yml of `metricbeat` at that time.